provider/resource.go

```diff
@@ -1311,6 +1318,7 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_key_vault_access_policy":           {Tok: azureResource(azureKeyVault, "AccessPolicy")},
 			"azurerm_key_vault_key":                     {Tok: azureResource(azureKeyVault, "Key")},
 			"azurerm_key_vault_secret":                  {Tok: azureResource(azureKeyVault, "Secret")},
+			"azurerm_key_vault_certificate_contacts":    {Tok: azureResource(azureKeyVault, "CertificateContacts")},
```

Comment:
nit: we generally [Add to the bottom of the relevant section by convention](https://github.com/pulumi/platform-providers-team/blob/main/playbooks/tf-provider-updating.md)